File: Android_App/app/src/main/java/uk/ac/aber/dcs/cascade/ui/database/enitity/Scheme.kt
package uk.ac.aber.dcs.cascade.ui.database.enitity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * @author Philip
 */
@Entity
data class Scheme(
    @PrimaryKey @ColumnInfo(name = "scheme_id") val schemeId: String,
    val title: String,
    val award: String,
    val department: String,
    @ColumnInfo(name = "undergraduate_or_postgraduate") val underOrPost: String,
    @ColumnInfo(name = "scheme_type") val schemeType: String,
    val year: String,
    val duration: Int,
    val url: String
)

File: AberCascadeCalulator/Models/Scheme.cs
using System;
using System.Collections.Generic;

namespace AberCascadeCalulator.Models
{
    public class Scheme
    {
        public string SchemeId { get; set; }
        public string Title { get; set; }
        public string Award { get; set; }
        public string Department { get; set; }
        public string UndergraduateOrPostgraduate { get; set; }
        public string SchemeType { get; set; }
        public long? Year { get; set; }
        public long? Duration { get; set; }
        public string Url { get; set; }
    }
}
